test/e2e/common_suite_test.go

```diff
@@ -55,3 +59,149 @@ func doTestCreateSimplePod(t *testing.T, assert CloudAssert) {
 		}).Feature()
 	testEnv.Test(t, simplePodFeature)
 }
+
+func doTestCreatePodWithConfigMap(t *testing.T, assert CloudAssert) {
+	// TODO: generate me.
+	namespace := "default"
+	name := "configmap-pod"
+	configmapname := "nginx-config"
+	configmapData := map[string]string{"example.txt": "Hello, world"}
+	containerName := "nginx"
+	pod := newPodWithConfigMap(namespace, name, containerName, "kata", configmapname)
+	configmap := newConfigMap(namespace, configmapname, configmapData)
+	nginxPodFeature := features.New("Configmap Pod").
+		WithSetup("Create pod", func(ctx context.Context, t *testing.T, cfg *envconf.Config) context.Context {
+			client, err := cfg.NewClient()
+			if err != nil {
+				t.Fatal(err)
+			}
+
+			if err = client.Resources().Create(ctx, configmap); err != nil {
+				t.Fatal(err)
+			}
+			if err = client.Resources().Create(ctx, pod); err != nil {
+				t.Fatal(err)
+			}
+			if err = wait.For(conditions.New(client.Resources()).PodRunning(pod), wait.WithTimeout(WAIT_POD_RUNNING_TIMEOUT)); err != nil {
+				t.Fatal(err)
+			}
+
+			return ctx
+		}).
+		Assess("Configmap is created and contains data", func(ctx context.Context, t *testing.T, cfg *envconf.Config) context.Context {
+			var podlist v1.PodList
+			var stdout, stderr bytes.Buffer
+			if err := cfg.Client().Resources(namespace).List(context.TODO(), &podlist); err != nil {
+				t.Fatal(err)
+			}
+			for _, i := range podlist.Items {
+				if i.ObjectMeta.Name == name {
+					if err := cfg.Client().Resources(namespace).ExecInPod(ctx, namespace, name, containerName, []string{"cat", "/etc/config/example.txt"}, &stdout, &stderr); err != nil {
+						t.Log(stderr.String())
+						t.Fatal(err)
+					}
+				}
+			}
+			if stdout.String() == "Hello, world" {
+				log.Infof("Data Inside Configmap: %s", stdout.String())
+				return ctx
+			} else {
+				t.Errorf("Configmap with invalid Data")
```

Comment:
```suggestion
				t.Errorf("Configmap with invalid Data: %s", stdout.String())
```
Can we add the string to the error for debug purposes